### DISCUSSION TITLE
👏欢迎来到诗翔自习讨论室

### DISCUSSION THREAD

**Discussion by ShixiangWang** [ShixiangWang/self-study#1] [General]
## 👋 欢迎！

这里是一个与诗翔讨论任何事情（主要技术和科研）的地方。我希望你在这里：

- 询问关心的问题
- 分享想法
- 与其他参与的社区成员互动
- 对自己和他人保持友好

王诗翔

2021.01

**Comment by zhangqt23** [comment]
你好！我想请教用您开的doabsolute R包进行多样本处理，怎么将同一样本的segmentfile和maf文件一一匹配呢？我有来自三十个病人的配对肿瘤和正常组织样本，现在我将sequenza得到的所有患者的segment.txt合并为一个seg，首列为sample,然后我合并了所有病人的maf文件成一个maf，首列为SampleId, absolute的必须字段包括Tumor_Sample_Barcode,全部是unknown。然后我运行doabsolute，返回无法识别每一个病人对应的maf,光用segment文件在跑了，我想是不是应该把Tumor_Sample_Barcode改成病人ID？这样就能识别对应的seg和maf了？

**Reply by ShixiangWang** [reply]
MAF 文件通过 Tumor_Sample_Barcode 识别样本 ID，seg 文件一般是 sample/Sample。具体请看 DoAbsolute README 的例子。

**Comment by Yunuuuu** [comment]
王老师，能否尝试一下[ggalign](https://github.com/Yunuuuu/ggalign)这个包，可以用ggplot2画跟ComplexHeatmap一样复杂的热图. 

![image](https://github.com/user-attachments/assets/02d2fe99-194e-4352-bc50-a94ef96c9409)

**Reply by ShixiangWang** [reply]
谢谢分享，我现在weekly做一个推荐

**Reply by Yunuuuu** [reply]
谢谢王老师
